src/main/resources/templates/interest/page.html

```diff
@@ -66,7 +66,12 @@ <h2>Places</h2>
                    th:href="@{/interests/{intId}/places/{placeId}(intId=${interest.id}, placeId=${place.id})}"
                 >
                     <div class="interest-place-img">
-                        <img src="https://picsum.photos/400/300" alt="placeholder"/>
+
+                        <img th:if="${place.place().imageNames.isEmpty()}" src="https://picsum.photos/400/300"

```

Comment:
Place.place().imageNames does not work anymore in latest develop branch after changes to PlaceInfoDTO.

List<String> imageNames needs to be added to PlaceInfoDTO for this to work.

Reply:
Update PlaceInfoDTO and test using it